test/App.test.jsx

```diff
@@ -0,0 +1,72 @@
+import { render, cleanup } from '@testing-library/react';
+import React from 'react';
+import { expect, it, vi, afterEach, beforeEach, describe } from 'vitest';
+import { ThemeProvider } from '@mui/material/styles';
+import { useSession, SessionProvider } from '@inrupt/solid-ui-react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import flushPromises from './testHelpers';
+
+import theme from '../src/theme';
+
+import App from '../src/App';
+
+const AppWithContexts = () => (
+  <Router>
+    <SessionProvider>
+      <ThemeProvider theme={theme}>
+        <App />
+      </ThemeProvider>
+    </SessionProvider>
+  </Router>
+);
+
+vi.mock('@inrupt/solid-client');
+
+vi.mock('@inrupt/solid-ui-react', async () => {
+  const lib = await vi.importActual('@inrupt/solid-ui-react');
```

Comment:
Need to mock `useSession` (its data is initialized by network requests), but don't want to replace the session provider or other components.